website/docs/hms.md

```diff
@@ -127,6 +127,16 @@ cd $HUDI_HOME/hudi-sync/hudi-hive-sync
 --table <tableName>
 ```
 
+```shell md title="example for s3 with HMS"
+./run_sync_tool.sh  \
+--metastore-uris 'thrift://hive-metastore:9083' \
+--partitioned-by city \
+--base-path 's3a://warehouse/people' \
+--database hudi_db \
```

Comment:
Parameterise these similar to the jdbc command ?

Reply:
This was specific to the new docker demo with S3.  The problem I found was that people needed a working example and not a syntax example.